src/applications/disability-benefits/all-claims/components/Autocomplete.jsx

```diff
@@ -79,9 +106,14 @@ const Autocomplete = ({
     setActiveIndex(index);
 
     const activeResult = resultsRef.current[index];
-    activeResult?.scrollIntoView({
-      block: 'nearest',
-    });
+
+    if (!isElementVisible(activeResult, listRef.current)) {
+      activeResult?.scrollIntoView({
```

Comment:
same here

Reply:
and anywhere else we're scrolling, just in case

Reply:
If just using `instant` then yup we wouldn't need to check for `prefers-reduced-motion` 👍